public/app/features/explore/LiveLogs.tsx

```diff
@@ -82,24 +152,33 @@ class LiveLogs extends PureComponent<Props> {
           <div
             ref={element => {
               this.liveEndDiv = element;
-              if (this.liveEndDiv) {
+              // This is triggered on every update so on every new row. It keeps the view scrolled at the bottom by
+              // default.
+              if (this.liveEndDiv && !isPaused) {
```

Comment:
Isnt this done already in cdu?

Reply:
Not really, the CDU part is to keep the user scrolled in the correct position when we pause and add 900 more rows to the view. So that only applies when user pauses and want to keep the position at the bottom of the container.